bower.json

```diff
@@ -7,6 +7,7 @@
   "dependencies": {
     "chart.js": "https://github.com/nnnick/Chart.js.git#2.0.0-beta2",
     "fontawesome": "4.5.0",
+    "fullcalendar": "2.6.0",
```

Comment:
Oops- Sorry this is my first time working a on big software project with
lots of ui libraries and this 'bower' tool.

*This is great learning opportunity for me! Could you send me the reference
you used to learn about bower and/or a quick-start guide on how I should be
using it to add ui libraries?*

Thanks, Wesley

On Wed, Feb 3, 2016 at 3:26 PM, Christian Murphy <notifications@github.com>
wrote:

> In bower.json
> <https://github.com/IMOD-ASU/imod/pull/673#discussion_r51797987>:
>
> > @@ -7,6 +7,7 @@
> >    "dependencies": {
> >      "chart.js": "https://github.com/nnnick/Chart.js.git#2.0.0-beta2",
> >      "fontawesome": "4.5.0",
> > +    "fullcalendar": "2.6.0",
>
> @WesCoomber <https://github.com/WesCoomber> when adding ui libraries
> please add them using bower.
> This keeps the libraries and the project source code separate.
> And keeps this diffs easy to read. [image: :smile:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/IMOD-ASU/imod/pull/673/files#r51797987>.
>